File: rei-do-chopp-api/ReiDoChopp.DataTransfer/Users/Requests/UserResetPasswordRequest.cs
using System.ComponentModel.DataAnnotations;

namespace ReiDoChopp.DataTransfer.Users.Requests
{
    public class UserResetPasswordRequest
    {
        [Required]
        public string Email { get; set; }

        [Required]
        public string Token { get; set; }

        [Required]
        public string Password { get; set; }

        [Required]
        public string CheckPassword { get; set; }
    }
}

File: rei-do-chopp-site/src/app/users/models/requests/user-reset-password.request.ts
export class UserResetPasswordRequest {
  Email: string;
  Token: string;
  Password: string;
  CheckPassword: string;

  constructor(params: Partial<UserResetPasswordRequest> = {}) {
    this.Email = params.Email ?? "";
    this.Token = params.Token ?? '';
    this.Password = params.Password ?? '';
    this.CheckPassword = params.CheckPassword ?? '';
  }
}
